ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -660,7 +660,11 @@ open class BottomCommandingController: UIViewController {
                 item.action?(item)
             }
         } else {
-            cell.setup(title: item.title ?? "", customView: iconView)
+            if let badge = item.badge {
```

Comment:
i don't think we should make specifics about badge. tomorrow it might be avatar group or checkbox etc. we need to make it more generic in my opinion